.github/workflows/all_solutions.yml

```diff
@@ -588,8 +601,8 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@03bee3930647ebbf994244c21ddbc0d4933aab4f # v2.3.0
         with:
-          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
-
+          egress-policy: audit
```

Comment:
Is this intentionally being left on audit?

Reply:
It is; this job is currently disabled, so until we run it a few times, we can't use any other policy.